### PR TITLE
Issue/http log policy

### PR DESCRIPTION
**Pull request by mohsha-msft**
Parent PR: https://github.com/Azure/azure-storage-blob-go/pull/217

### CODE REVIEW

**Comment by mohsha-msft** [comment]
@pharaujo 

Can you please fix whatever is breaking from your changes? Reach out if you want more information or you get stuck somewhere. We really appreciate your contribution.

**Comment by pharaujo** [comment]
@mohsha-msft looking.

**Comment by pharaujo** [comment]
@mohsha-msft
I think I fixed the issue, but now I'm a bit confused as to where should I push the changes to. Is it to my original branch and then you merge it to this branch? Something else?

**Comment by mohsha-msft** [comment]
> @mohsha-msft
> I think I fixed the issue, but now I'm a bit confused as to where should I push the changes to. Is it to my original branch and then you merge it to this branch? Something else?

Great! If you've done the changes, raise a PR to this branch i.e. ``issue/http-log-policy``. I'll pull the changes in the PR.

**Comment by pharaujo** [comment]
@mohsha-msft 
Done in #229

**Comment by mohsha-msft** [comment]
@pharaujo 

Did you double check by running the test cases locally which were failing in the previous build? If those are running fine in local, I'll go ahead and merge it with this PR to trigger the complete test.

**Comment by pharaujo** [comment]
@mohsha-msft I just created a test storage account and am running the tests locally now.

**Comment by pharaujo** [comment]
@mohsha-msft The panics are gone, but there are still some test failures. I ran the test suite in the `dev` branch without my changes and it has the same failures. I'm fairly certain the failing tests are not from my changes. Can you assess?

**Comment by mohsha-msft** [comment]
Looks good to me. PR needs second review.

File: azblob/zc_policy_request_log.go
Comment (zezha-msft) [inline azblob/zc_policy_request_log.go:78]: minor: no need for the else

**Review by zezha-msft — APPROVED** [review]
lgtm

File: azblob/zc_policy_request_log.go
Comment (mohsha-msft) [reply azblob/zc_policy_request_log.go:78]: Thanks for the review. Sure will fix it.

**Pull request merged**
